src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
```

Comment:
A subscription request comes in, the q-string is parsed and stored in the subCache.
Or, what is taken from mongo at startup (subscriptions) are parsed and stored in the cache.
TriggeredSubscriptions are esat to point to the parsed 'q expression' and the match method and mongoFilters vector is used by mongoBackend.